sdk/daml-script/test/daml/upgrades/UpgradesTest.daml

```diff
@@ -205,12 +210,36 @@ exerciseV1Util p c = p `submit` createAndExerciseExactCmd (V1.ValidUpgradeUtils
 exerciseV2Util : Choice V2.ValidUpgradeUtils c r => Party -> c -> Script r
 exerciseV2Util p c = p `submit` createAndExerciseExactCmd (V2.ValidUpgradeUtils with party = p) c
 
+tryExerciseV0Util : Choice V0.ValidUpgradeUtils c r => Party -> c -> Script (Either SubmitError r)
+tryExerciseV0Util p c = p `trySubmit` createAndExerciseExactCmd (V0.ValidUpgradeUtils with party = p) c
+
 tryExerciseV1Util : Choice V1.ValidUpgradeUtils c r => Party -> c -> Script (Either SubmitError r)
 tryExerciseV1Util p c = p `trySubmit` createAndExerciseExactCmd (V1.ValidUpgradeUtils with party = p) c
 
 tryExerciseV2Util : Choice V2.ValidUpgradeUtils c r => Party -> c -> Script (Either SubmitError r)
 tryExerciseV2Util p c = p `trySubmit` createAndExerciseExactCmd (V2.ValidUpgradeUtils with party = p) c
 
+mustBeWronglyTypedContract (Left (WronglyTypedContract {})) = pure ()
+mustBeWronglyTypedContract  _ = assertFail "Wrong"
+
+fetchNonUpgradableFail : Script ()
+fetchNonUpgradableFail = do
```

Comment:
I think mentioning that this is an LF version upgrade/downgrade in the test name would make this a little more clear.
Its hard to see what the test means from the script name